### ISSUE TITLE
Implement PEP

### ISSUE THREAD

**Issue by JPRuehmann** [mailvelope/mailvelope#464]
Imlementing PEP (Pretty Easy Privacy) would be a big help.
Since it does the same thing as PGP/GPG but is much easier.
Thanks,

**Comment by yoshimo** [comment]
@toberndo , is this something you are interested in?

**Comment by JPRuehmann** [comment]
Yes, because I like to use Tine 2.0 but to use Crypto with it I need mailvelope.
And since Crypto is unusable for normal people the normal way, I think pep would be a big step to make Crypto usable by the Masses.
Thanks,

**Comment by mdik** [comment]
How would you imagine this to work?

Please see also my comment on issue https://github.com/mailvelope/mailvelope/issues/437

**Comment by JPRuehmann** [comment]
I don´t know, like Enigmail?
There is a Webpage for PeP on that the developers can be contacted.
I think they can help you.

**Comment by szaimen** [comment]
Any update here?
